src/ng/q.js

```diff
@@ -376,6 +378,7 @@ function qFactory(nextTick, exceptionHandler, errorOnUnhandledRejections) {
   }
 
   function processChecks() {
+    // eslint-disable-next-line no-unmodified-loop-condition
```

Comment:
Interesting. According to [this][1]:
> If a reference is inside of a binary expression or a ternary expression, this rule checks the result of the expression instead.

(And our expression should qualify as "binary expression".)

Nevermind, if it does complain, there's not much we can do :smile:

[1]: http://eslint.org/docs/rules/no-unmodified-loop-condition